src/main/java/com/gregtechceu/gtceu/data/recipe/configurable/RecipeAddition.java

```diff
@@ -691,6 +724,8 @@ private static void hardRedstoneRecipes(Consumer<FinishedRecipe> provider) {
                     .save(provider);
             ASSEMBLER_RECIPES.recipeBuilder("jukebox").duration(100).EUt(16).inputItems(ItemTags.PLANKS, 8)
                     .inputItems(gem, Diamond).outputItems(new ItemStack(Blocks.JUKEBOX)).save(provider);
+            ASSEMBLER_RECIPES.recipeBuilder("target").duration(100).EUt(16).inputItems(Items.REDSTONE, 4)
+                    .inputItems(Items.HAY_BLOCK).outputItems(new ItemStack(Blocks.TARGET)).save(provider);
```

Comment:
why is this duplicated from line 635?

Reply:
635 needs to be replaced with the hard mode recipe once I think of it